tests/test_lib_policydecorator.py

```diff
@@ -58,7 +58,12 @@ def fake_check_otp(dummy, pin, user=None, options=None):
         return pin == "FAKE"
 
     @staticmethod
-    def fake_check_token_list(tokenobject_list, passw, user=None, options=None, allow_reset_all_tokens=True,
+    def fake_check_token_list_reset(tokenobject_list, passw, user=None, options=None, allow_reset_all_tokens=True,
+                              result=False):
+        return result, "some text"    \
+
+    @staticmethod
+    def fake_check_token_list_increase(tokenobject_list, passw, user=None, options=None, allow_increase_failcount=True,
```

Comment:
You are not using this method.
But worst of all you are not testing the increase at all.

Please ensure, that you test the functionality in the `check_token_list` (line 2390)

Reply:
# You are not using this method.
fixed in a79404d8c496e829974bb3149e58145465639236.